### PR TITLE
add "UraMasyu" module's manual, PDF, and json

### PR DESCRIPTION
**Pull request by YY-otter**

### CODE REVIEW

File: HTML/UraMasyu translated (日本語 — 裏ましゅ) (Y.Y.).html [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+                    <text transform="translate(700 2098.63)"><tspan x="0" y="0" class="st5">Pass straight through black</tspan><tspan x="0" y="52.8" class="st5">circles, and turn in one or</tspan><tspan x="0" y="105.6" class="st5">both cells on either end.</tspan></text>
+                    <text transform="translate(1757.15 2098.63)"><tspan x="0" y="0" class="st5">No turning in black circles</tspan><tspan x="0" y="52.8" class="st5">or passing straight through</tspan><tspan x="0" y="105.6" class="st5">both cells on either end.</tspan></text>
+                    <text transform="translate(701 2994.68)"><tspan x="0" y="0" class="st5">The path must be a single,</tspan><tspan x="0" y="52.8" class="st5">continuous loop that passes</tspan><tspan x="0" y="105.6" class="st5">through all of the circles.</tspan></text>
+                    <text transform="translate(1757 2994.68)"><tspan x="0" y="0" class="st5">No small loops (top left),</tspan><tspan x="0" y="52.8" class="st5">spurrs (bottom left), or</tspan><tspan x="0" y="105.6" class="st5">trapped ends (right).</tspan></text>
```
Comment (Luminoscity) [inline HTML/UraMasyu translated (日本語 — 裏ましゅ) (Y.Y.).html:222]: Making an imported SVG with lots of text that has been converted to <path> is not acceptable as it cannot be translated easily, and such path conversions take up hundreds more kilobytes than pure text. So I converted your diagrams to be embedded in the HTML and have actual text. Now you just have to type in the Japanese text you want here.

**Review by Luminoscity — CHANGES_REQUESTED** [review]
embedded diagrams for translatability
fix linting errors (in the future, make sure you are checking with a DM to Logbot that your HTML files pass linting)
fixed formatting issues
add button that makes the page readable
delete duplicate SVG
PDFs are automatically generated and should not be included
fixed JSON errors

**Comment by YY-otter** [comment]
The UraMasyu module has been updated to align with this content.

**Comment by Luminoscity** [comment]
Next time, you don't need to keep pushing merge commits to your fork.
Now that it is merged, please rebase your fork to the latest so future pull requests won't have past commits in them.
Nice job on this mod.

**Pull request merged**